stubs/pynput/pynput/mouse/_base.pyi

```diff
@@ -0,0 +1,38 @@
+import enum
+from _typeshed import Self
+from types import TracebackType
+from typing import Any, Callable
+
+from pynput._util import AbstractListener
+
+class Button(enum.Enum):
+    unknown: int
+    left: int
+    middle: int
+    right: int
+
+class Controller:
+    def __init__(self) -> None: ...
+    @property
+    def position(self) -> tuple[int, int]: ...
+    @position.setter
+    def position(self, pos: tuple[int, int]) -> None: ...
```

Comment:
Just as a heads up: This might not work and we might need to use just `position: tuple[int, int]`, but we can always change it later if that turns out to be problematic.